components/Card/CardTitle.tsx

```diff
@@ -27,93 +13,151 @@ interface CardTitleProps {
-          <CardBootstrap.Text className={`fs-5 lh-sm mb-1`}>
-            {subheader}
-          </CardBootstrap.Text>
-        )}
-        {timestamp && (
-          <CardBootstrap.Text className={`fs-6 lh-sm`}>
-            {timestamp}
-          </CardBootstrap.Text>
-        )}
+      <CardBootstrap.Body className="px-3 py-0">
+        <CardTitleHeadline
+          court={court}
+          header={header}
+          subheader={subheader}
+          type={type}
+        />
+        <CardTitleFollowing
+          header={header}
+          subheader={subheader}
+          isBillMatch={isBillMatch}
+          isUserMatch={isUserMatch}
+          type={type}
+        />
       </CardBootstrap.Body>
-      {inHeaderElement && inHeaderElement}
     </CardBootstrap.Body>
   )
 }
 
-// newsfeed bill card title
-export const CardTitleV2 = (props: CardTitleProps) => {
-  const {
-    court,
-    header,
-    subheader,
-    timestamp,
-    imgSrc,
-    imgTitle,
-    inHeaderElement,
-    isBillMatch
-  } = props
+const CardTitleHeadline = (props: CardTitleProps) => {
+  const { court, header, subheader, type } = props
   const { t } = useTranslation("common")
 
-  return (
-    <CardBootstrap.Body className={`align-items-center d-flex px-2 pt-2 pb-0`}>
-      <div className="justify-content-middle d-flex flex-column align-items-center">
-        {imgSrc && <img alt="" src={imgSrc} width="32" height="32" />}
-      </div>
-      <CardBootstrap.Body className="px-3 py-0">
+  switch (type) {
+    case "testimony":
+      return (
+        <>
+          {header && subheader && (
+            <CardBootstrap.Title
+              className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+            >
+              {subheader} {t("newsfeed.endorsed")}
+              <a href={`/bills/${court}/${header}`}>
+                <strong>{formatBillId(header)}</strong>
+              </a>
+            </CardBootstrap.Title>
+          )}
+        </>
+      )
+    case "bill":
+      return (
+        <>
+          {header && (
+            <CardBootstrap.Title
+              className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+            >
+              <a href={`/bills/${court}/${header}`}>
+                <strong>{formatBillId(header)}</strong>
+              </a>{" "}
+              {subheader && (
+                <>
+                  {t("newsfeed.action_update")}
+                  {subheader}
+                </>
+              )}
+            </CardBootstrap.Title>
+          )}
+        </>
+      )
+    default:
+      return (
+        <CardBootstrap.Title
+          className={`align-items-start fs-6 lh-sm mb-1 text-secondary`}
+        >
+          <strong>{header}</strong>
+        </CardBootstrap.Title>
+      )
+  }
+}
+
+const CardTitleFollowing = (props: CardTitleProps) => {
+  const { header, subheader, isBillMatch, isUserMatch, type } = props
+  const { t } = useTranslation("common")
+
+  if (type == `no results`) {
```

Comment:
Instead of having a separate card type for "No Results", could we handle that case at the list level (so we either show a "No Results" empty state or a List of Newsfeed card results)?

Side note - it doesn't look like the design has an empty state - let's make a note to call that out in design reviews for future features.

Reply:
Outdated -> I think I changed `no results` to `` in the process of moving somethings around